connectors/pinecone-sink/pinecone-sink.yaml

```diff
@@ -0,0 +1,4 @@
+repository: https://github.com/streamnative/pulsar-io-pinecone
+icon: "/images/connectors/pinecone-logo.png"
+sn_available: true
```

Comment:
Before the sn cloud, this value should be false.

Reply:
Can you clarify what you mean by that? I do not understand

Reply:
This value serves as a label on the [pulsar-hub](https://docs.streamnative.io/hub). If it's true, it means it's built-in on our SN Cloud.


pulsar-hub:
<img width="2395" alt="image" src="https://github.com/streamnative/pulsar-hub/assets/33416836/9409bd21-32c7-48b9-866f-a1fb4382a815">

sn-cloud:
<img width="1460" alt="image" src="https://github.com/streamnative/pulsar-hub/assets/33416836/e7168297-c21c-4a0f-bc16-a2d1f80bf17a">

Reply:
I will be adding it soon